dbt/adapters/redshift/relation_configs/materialized_view.py

```diff
@@ -169,6 +169,15 @@ def parse_relation_results(cls, relation_results: RelationResults) -> Dict:
                     "query": agate.Table(
                         agate.Row({"definition": "<query>")}
                     ),
+                    "column_descriptor": agate.Table(
+                        agate.Row({
+                            "schema": "<schema_name>",
```

Comment:
Since this is coming in alongside the relation config, `schema` and `table` are unnecessary. Let's simplify this to `column`, `is_sort_key`, and `is_dist_key`.

We should also update `is_sort_key` to the sort key order since swapping the order of the columns does create a different sort key, hence should be considered in the configuration changes.

Reply:
fixed